doc/authority.md

```diff
@@ -59,4 +59,40 @@ If you do not configure a template, the default is to use the `cred-id` and `aut
     "objectType":"Agent"
 }
 ```
+
+### OIDC Authority
+
+When SQL LRS is [configured for OIDC](oidc.md) an alternative template is used that forms the authority from token claims:
+
+``` json
+{
+  "objectType": "Group",
+  "member": [
+    {
+      "account": {
+        "homePage": "{{iss}}",
+        "name": "{{lrsql/resolved-client-id}}"
+      }
+    },
+    {
+      "account": {
+        "homePage": "{{iss}}",
+        "name": "{{sub}}"
+      }
+    }
+  ]
+}
+```
+
+Possible claims vary by identity provider, see the [comprehensive list here](https://www.iana.org/assignments/jwt/jwt.xhtml). SQL LRS provides one additional value, `lrsql/resolved-client-id` that resolves to the following claims in order of precedence:
+
+1. `client_id`
+2. `azp`
+3. `aud` (if `aud` is a string)
+4. `aud[0]` (if `aud` is an array)
```

Comment:
The link I give above the example has an exhaustive list. Anyone using this at the level of sophistication where they need to use this will be able to go figure out what they are.